german/Leichen/Leichen.md

```diff
@@ -3,4 +3,6 @@ völlig durchlöcherten Leichnamen neben sich, deren Blut sie mittlerweile gross
 Lebewesen im See gespendet haben. Ihr Boot ist bereits in leichter Schieflage und hastig
 werfen die beiden Leichen aus dem Boot.
 
-Sie schnaufen einmal kräftig durch während ihnen tausend Gedanken durch den Kopf gehen...
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.